rviz_rendering/CMakeLists.txt

```diff
@@ -142,6 +151,16 @@ if(BUILD_TESTING)
     ament_cpplint()
     ament_lint_cmake()
     ament_uncrustify()
+
+    ament_add_gtest(test_rviz_rendering src/test/string_helper_test.cpp)
+    if(TARGET test_rviz_rendering)
+        target_include_directories(test_rviz_rendering PUBLIC src/rviz_rendering)
+        target_link_libraries(test_rviz_rendering ${PROJECT_NAME})
+    endif()
 endif()
 
-ament_package()
+list(APPEND ${PROJECT_NAME}_CONFIG_EXTRAS
+        "${CMAKE_CURRENT_SOURCE_DIR}/src/cmake/register_rviz_ogre_media_exports_hook-extras.cmake"
+        "${CMAKE_CURRENT_SOURCE_DIR}/src/cmake/register_rviz_ogre_media_exports.cmake")
+
+ament_package()
```

Comment:
Please make sure all the files have a newline at the end of the file (again the C++ style checker would catch this). You can see it on github like this:

<img width="170" alt="screen shot 2017-10-25 at 8 24 46 pm" src="https://user-images.githubusercontent.com/100427/32033651-90d74132-b9c2-11e7-8220-1bbe187ac360.png">